content/assignments/hello-world/start-coding.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Task 3: Start Coding"
+date: 2023-01-05T09:57:32-06:00
+draft: false
+weight: 3
+originalAuthor: Courtney  # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: Kimberly # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: Courtney # update any time edits are made after review
+lastEditorGitHub: speudusa # update any time edits are made after review
+lastMod: copy edits and links for the debugger added, created separate page for this section
+---
+
+##  Your Task: 
```

Comment:
I find this heading odd, what about,
```suggestion
##  Update the `SayHello()` Method: 
```

Reply:
What if I drop the title in this section and they dive straight into the task?